client/src/TenderClient.js

```diff
@@ -0,0 +1,78 @@
+import axios from "axios";
+
+class TenderClient {
+	constructor(baseURL, token) {
+		this.client = axios.create({
+			baseURL: baseURL,
+			headers: {
+				Authorization: `Bearer ${token}`,
```

Comment:
The token should be retrieved from local storage every time we try to make a request, since the token might change if the user signs-out and signs-in again